pages/team_management/permissions.md

```diff
@@ -34,6 +34,9 @@ A user who is a _Buildkite organization administrator_ can access the [**Organiz
 
     * Create a new team, using the **New Team** button.
     * Administer (with full control) the [team-](#manage-teams-and-permissions-team-level-permissions), [pipeline-](#manage-teams-and-permissions-pipeline-level-permissions), [test suite-](/docs/test-analytics/permissions#manage-teams-and-permissions-test-suite-level-permissions) and [registry-](/docs/packages/permissions#manage-teams-and-permissions-registry-level-permissions)level settings throughout their Buildkite organization.
+
+        **Note:** Registry-level settings are only available once [Buildkite Packages has been enabled](/docs/packages/permissions#enabling-buildkite-packages).
```

Comment:
Worth making this easier to link to 🕺🏻

Reply:
Creating that new 'Enabling Buildkite Packages' sub-heading was definitely a great idea! Thanks @mbelton-buildkite for that